scalameta/common/shared/src/main/scala/org/scalameta/internal/MacroHelpers.scala

```diff
@@ -70,6 +70,7 @@ trait MacroHelpers extends DebugFinder
   lazy val IndexOutOfBoundsException = hygienicRef[IndexOutOfBoundsException]
   lazy val IteratorClass = tq"_root_.scala.collection.Iterator"
   lazy val SeqClass = tq"_root_.scala.collection.immutable.Seq"
```

Comment:
How about we return an `immutable.Seq` for consistency with `children`? It'll likely change to List with the re-organization PR.